### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Muhammad-Mubarak**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,26 +17,30 @@ gem 'delayed_job'
...
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem "better_errors"
   gem "binding_of_caller"
 end
 
-group :production do
-  gem 'rails_12factor'
-  gem 'pg'
-end
+# group :production do
```
Comment (WaqarHassan) [inline Gemfile:40]: un comment

File: app/datatables/application_datatable.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class ApplicationDatatable < AjaxDatatablesRails::ActiveRecord
```
Comment (WaqarHassan) [inline app/datatables/application_datatable.rb:1]: remove file

File: db/schema.rb [hunk 3]
```diff
@@ -12,12 +12,15 @@
...
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
   create_table "companies", force: :cascade do |t|
     t.string   "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "deleted_at"
-    t.index ["deleted_at"], name: "index_companies_on_deleted_at"
+    t.index ["deleted_at"], name: "index_companies_on_deleted_at", using: :btree
```
Comment (WaqarHassan) [inline db/schema.rb:23]: remove using: btree

File: config/database.yml [hunk 4]
```diff
@@ -1,15 +1,17 @@
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
```
Comment (WaqarHassan) [inline config/database.yml:2]: remove database.ymml

File: config/initializers/wicked_pdf.rb [hunk 5]
```diff
@@ -11,7 +11,8 @@
 WickedPdf.config = {
```
Comment (WaqarHassan) [inline config/initializers/wicked_pdf.rb:11]: revert this file

**Review by WaqarHassan — CHANGES_REQUESTED** [review]
Invoices list should be in descending order

File: db/schema.rb (hunk 3 above)
Comment (Muhammad-Mubarak) [reply db/schema.rb:23]: I think this comes as a result of dumping the database so will not cause any problem.

**Review by WaqarHassan — APPROVED** [review]

**Pull request merged**